docs: Improve the README (#10490)
CloudFire-LLC/cloudfire: docs/README.md

  </picture>
</p>
<p align="center">

>  <strong>Secure remote access that's 3x faster than OpenVPN with zero-trust, peer-to-peer connections</strong>

Reviewer: I really liked the old catch-phrase; probably because it was simple. Perhaps we could iterate on it further? 

"A modern, faster and more secure alternative to legacy VPNs" ? wdyt?

Author: This line probably matters quite a bit for SEO. I'll need to check our marketing analytics and see if we can condense it while still hitting the right keywords.

</p>

<p align="center">
  <a href="https://www.firezone.dev/kb">Docs</a>
  | <a href="https://www.firezone.dev/kb/quickstart">Quickstart</a>
  | <a href="https://www.firezone.dev/kb/client-apps">Download Clients</a>

>   | <a href="https://discourse.firez.one/?utm_source=readme">Community</a>

Author: ```suggestion
  | <a href="https://github.com/firezone/firezone/discussions">Community Forums</a>
```

  | <a href="https://discourse.firez.one/?utm_source=readme">Community</a>
  | <a href="https://www.firezone.dev/support">Support</a>
</p>

---

<p align="center">
  <a href="https://github.com/firezone/firezone/releases">
    <img src="https://img.shields.io/github/v/release/firezone/firezone?color=%23999">
  </a>
  <a href="https://discourse.firez.one/?utm_source=readme">
    <img src="https://img.shields.io/static/v1?logo=discourse&logoColor=959DA5&label=support%20forum&labelColor=333a41&message=join&color=611f69" alt="firezone Discourse" />

>   </a>

Reviewer: YES! 

- the release would be good if we had unified version scheme, otherwise it was just confusing to me as a newcomer why we advertised android client
- discourse forum didn't bring any value

</p>
-->

## Getting Started

> ### Option 1: Cloud (Recommended)

Reviewer 2: There's no `Option 2` from what I saw.  Was there supposed to be?  If not, should we remove this header?

Reviewer: I agree, by providing docs to just the recommended solution we're making it simpler for everyone.

> ### Option 1: Cloud (Recommended)

Author: ```suggestion
```

- [kotlin/](../kotlin/android): Android / ChromeOS clients.
- [website/](../website): Marketing website and product documentation.

## License & Pricing

### Open Source (Apache 2.0 + Elastic 2.0)
- ✅ Full source code available for audit
- ✅ Self-hosting allowed (educational/hobby use)
- ✅ Community support via Discord and forums
- ⚠️ Production self-hosting not officially supported

### Cloud - Usage Based
- ✅ Managed hosting with SLA
- ✅ Production-ready with enterprise support
- ✅ Automatic updates and maintenance
- 💰 Starting free, then per-seat pricing
- [**View detailed pricing →**](https://www.firezone.dev/pricing?utm_source=readme)

**Pricing Overview:**
- **Starter:** Free for <!-- TODO: X --> users/month

> - **Team:** $<!-- TODO: X -->/user/month for advanced features

Reviewer: sneaky copilot, changing the rules of the starter plan ;-)

- [kotlin/](../kotlin/android): Android / ChromeOS clients.
- [website/](../website): Marketing website and product documentation.

## License & Pricing

> ### Open Source (Apache 2.0 + Elastic 2.0)

Reviewer: IMHO I would put the open source *after* the thing we actually want to encourage; the Firezone-managed plan.

I believe we could best achieve it by spitting "Licence" and "Pricing" - when I look into pricing, I want to know how much this could cost me; and when I just look up licence, I'm not much interested in paying anything.

- ✅ Full source code available for audit
- ✅ Self-hosting allowed (educational/hobby use)
- ✅ Community support via Discord and forums
- ⚠️ Production self-hosting not officially supported

### Cloud - Usage Based
- ✅ Managed hosting with SLA
- ✅ Production-ready with enterprise support
- ✅ Automatic updates and maintenance
- 💰 Starting free, then per-seat pricing
- [**View detailed pricing →**](https://www.firezone.dev/pricing?utm_source=readme)

**Pricing Overview:**

> - **Starter:** Free for <!-- TODO: X --> users/month

Author: ```suggestion
- **Starter:** Free for 6 users/month
```

> - **Team:** $<!-- TODO: X -->/user/month for advanced features

Author: ```suggestion
- **Team:** $5/user/month for advanced features
```

> - **Enterprise:** Custom pricing with SSO, compliance, priority support

Author: ```suggestion
- **Enterprise:** Custom pricing with SSO, compliance, priority support, and SLAs
```

### Enterprise Features

> - 🔐 **SSO/SAML** - Integrate with your identity provider

Author: ```suggestion
- 🔐 **SSO** - Integrate with any OIDC-compatible identity provider
```

> - 📝 **Audit Logs** - Complete activity tracking for compliance

Author: ```suggestion
- 🧑‍🧒‍🧒 **Directory Sync** - Sync users and groups from Google Workspace, Microsoft Entra, and Okta
```

- 🏢 **Priority Support** - Dedicated team for your organization
- 🎯 **Custom Integrations** - Tailored solutions for your infrastructure

## Frequently asked questions (FAQ)

> 

Reviewer: (not from this changeset):  would probably move the FAQ into its own file, and just link to it at the top, just like with KB. makes it easier to find when you have a question and easier to skip when you don't

Author: Good point, it might be a good idea to recreate this FAQ in the [discussion forums](https://github.com/firezone/firezone/discussions)

### Migrating from Other VPNs

**Coming from OpenVPN, ZeroTier, or Tailscale?**
<!-- TODO: Add specific migration guides -->
- [Migrate from OpenVPN →](<!-- TODO: Add migration guide link -->)
- [Migrate from ZeroTier →](<!-- TODO: Add migration guide link -->)
- [Migrate from Tailscale →](<!-- TODO: Add migration guide link -->)
- [General VPN Migration Guide →](<!-- TODO: Add general guide link -->)

**Migration typically involves:**
1. Installing Firezone alongside your current solution
2. Configuring equivalent access policies
3. Testing connectivity with a subset of users
4. Gradually migrating users and decommissioning old infrastructure

## Documentation

Additional documentation on general usage, troubleshooting, and configuration
can be found at [https://www.firezone.dev/kb](https://www.firezone.dev/kb).

## Join Our Community

### Quick Ways to Contribute
- ⭐ **Star this repo** to show support and stay updated
- 🐛 **Report bugs** or request features via [GitHub Issues](https://github.com/firezone/firezone/issues)
- 💬 **Join our [Discord](https://discord.gg/DY8gxpSgep)** for live discussions and community support

> - 🗣️ **Join the [Discussion Forums](https://discourse.firez.one/?utm_source=readme)** for longer-form conversations

Author: ```suggestion
- 🗣️ **Join the [Discussion Forums](https://github.com/firezone/firezone/discussions)** for longer-form conversations
```

### For Contributors
- 🎯 **Check out [good first issues](https://github.com/firezone/firezone/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)** to get started
- 📖 **Read our [contribution guide](CONTRIBUTING.md)** for development setup
- 🏗️ **See the [development environment setup](CONTRIBUTING.md)** to start coding
- 🔄 **Submit a pull request** - all contributions welcome!

**Recognition:** All contributors get <!-- TODO: Describe contributor recognition/rewards -->!

> 

Author: ```suggestion
<!-- TODO: Add paragraph here with instructions for starting a Discussion prior to investing a bunch of time in any user-facing or large changes to get buy-in. -->
```

## Get Help

**Community Support (Free)**
- [Discord Server](https://discord.gg/DY8gxpSgep) - Live chat with users and team

> - [Discussion Forums](https://discourse.firez.one/?utm_source=readme) - Detailed Q&A

Author: ```suggestion
- [Discussion Forums](https://github.com/firezone/firezone/discussions) - Detailed Q&A and new feature proposals
```

> - [GitHub Issues](https://github.com/firezone/firezone/issues) - Bug reports and feature requests

Author: ```suggestion
- [GitHub Issues](https://github.com/firezone/firezone/issues) - Bug reports and well-scoped items
```
